extension/persistence/relational-jdbc/src/main/resources/postgres/schema-v1-postgresql.sql

```diff
@@ -15,16 +15,12 @@
 -- KIND, either express or implied.  See the License for the
 -- specific language governing permissions and limitations
 -- under the License.
---
 
--- Note: Database and schema creation is not included in this script. Please create the database and
--- schema before running this script. for example in psql:
--- CREATE DATABASE polaris_db;
--- \c polaris_db
--- CREATE SCHEMA polaris_schema;
--- set search_path to polaris_schema;
+CREATE SCHEMA IF NOT EXISTS POLARIS_SCHEMA;
+SET search_path TO POLARIS_SCHEMA;
```

Comment:
`search_path` may be required on all connections (even when the schema already exists), right?

Reply:
yes, so we handle it with 2 part identifiers in the table name essentially like `<schema>.<table_name>` no need for connection string